src/api/stats.ts

```diff
@@ -96,6 +96,42 @@ const getTVL = async (assetId?: string) => {
   return data
 }
 
+export const useFee = (assetId?: string | "all") => {
+  const { assets } = useRpcProvider()
+
+  return useQuery(
+    QUERY_KEYS.fee(assetId),
+    assetId
+      ? async () => {
+          const asset_id = assetId === "all" ? undefined : assetId
+          const data = await geFee(asset_id)
+
+          if (assets.native.id === asset_id)
+            return { ...data[0], projected_apy_perc: 0 }
+
+          return data[0]
+        }
+      : undefinedNoop,
+    {
+      enabled: !!assetId,
+      refetchInterval: 60000,
+    },
+  )
+}
+
+const geFee = async (assetId?: string) => {
+  const res = await fetch(
+    `https://api.hydradx.io/hydradx-ui/v1/stats/fees/${
+      assetId != null ? `/${assetId}` : ""
```

Comment:
if you a are checking for `undefined`, why not write it like this? ```assetId ? `/${assetId}` : ""```

It is a good practice to avoid using non-strict equality checks `!=` and `==`, only use strict equality checks `!==` and `===`

Reply:
I won't use !=, but you should remember that HDX has 0 id, and in this case assetId ? will be falsy